### ISSUE TITLE
Pre-commit fail

### ISSUE THREAD

**Issue by vict0rsch** [mlco2/codecarbon#64]
see : https://github.com/mlco2/codecarbon/runs/1424483572

```
Error: Unable to process command '::set-env name=PY::d874f77ec1348dfde58a9317309117a9df04cdc975815e70e2910a54b56d97ae' successfully.
Error: The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands
```

Any idea @Lothiraldan ?

**Comment by Lothiraldan** [comment]
This has been fixed in another PR: https://github.com/mlco2/codecarbon/pull/63/files#diff-ac5eead3f3ce4863c524fff031a87b7aecccb4a0493df087a4e1c704a1505036

**Comment by vict0rsch** [comment]
I missed that great thanks let's wait for that one then

**Comment by vict0rsch** [comment]
Fixed in #69
